projects/kserve/visualizations/kserve-llm/models/kpi.py

```diff
@@ -56,7 +60,11 @@ def kserve_llm_load_test_tpot_p99(lts_payload):
 
 @matbench_models.KPIMetadata(help="All values of time to first token", unit="List[ms]")
 def kserve_llm_load_test_ttft(lts_payload):
-    return lts_payload.results.time_to_first_token.values
+    return [v for v in lts_payload.results.time_to_first_token.values if v is not None]
+
+@matbench_models.KPIMetadata(help="Number of failures reported in TTFT", unit="int")
+def kserve_llm_load_test_ttft_failures(lts_payload):
+    return sum([1 if v == None else 0 for v in lts_payload.results.time_to_first_token.values])
```

Comment:
similar simplification as above:
```suggestion
    return sum(1 for v in lts_payload.results.time_to_first_token.values if v is None)
```
(could even use `len` instead of `sum`, might be a tad faster, but I find `sum` more explicit here, and the speed gain is null for such small lists)